lib/tesla/multipart.ex

```diff
@@ -103,7 +103,7 @@ defmodule Tesla.Multipart do
         false -> headers
       end
 
-    data = File.stream!(path, [], 2048)
+    data = File.stream!(path, 2048)
```

Comment:
```suggestion
    data = stream_file!(path, 2048)
```

Then later on somewhere in the module

```elixir
if Version.compare(System.version(), "1.16.0") do
  defp stream_file!(path, bytes), do: File.stream!(path, bytes)
else
    defp stream_file!(path, bytes), do: File.stream!(path, [], bytes)
end

Reply:
Thank you @yordis!